state/azure/blobstorage/blobstorage_test.go

```diff
@@ -50,13 +50,11 @@ func TestGetBlobStorageMetaData(t *testing.T) {
 	t.Run("All parameters passed and parsed", func(t *testing.T) {
 		m := make(map[string]string)
 		m["accountName"] = "acc"
-		m["accountKey"] = "key"
```

Comment:
Why `key` is not parsed anymore?

Reply:
The `accountKey` property doesn't exist anymore in the object (see line 59 deleted in the test below).

Instead, the shared key is read in the method that returns the `azblob.Credential` object: https://github.com/dapr/components-contrib/blob/b41101713896855d113db2425d4b5163d633e8c5/authentication/azure/storage.go#L26